README.md

```diff
@@ -23,6 +23,25 @@ computers.
   </a>
 </div>
 
+## Example
+
+```js
+const start = new Date();
+const res = await fetch("https://ipfs.io/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/");
+console.log(`TTFB: ${new Date() - start}ms`);
+```
+
+```bash
+$ zinnia run example.js
+TTFB: 235ms
+```
+
```

Comment:
How about adding a link to [Building modules](https://github.com/filecoin-station/zinnia/blob/main/docs/building-modules.md) or [Platform APIs](https://github.com/filecoin-station/zinnia/blob/main/docs/building-modules.md#platform-apis), so that readers know when to continue their journey?